setup_cluster.sh

```diff
@@ -83,6 +83,7 @@ function deploy_operators() {
     NAME=host-operator OPERATOR_NAME=toolchain-host-operator NAMESPACE=$HOST_OPERATOR_NS envsubst < $PREVIOUS_DIR/config/operator_deploy.yaml | oc apply -f -
     make get-registration-service-repo login-as-admin
     make deploy-registration HOST_NS=$HOST_OPERATOR_NS REG_IMAGE_NAME=quay.io/codeready-toolchain/registration-service:v0.1
+    oc apply -f $PREVIOUS_DIR/config/reg_service_route.yaml
```

Comment:
what about making this call to `oc apply` after the `cd $PREVIOUS_DIR` below?

Reply:
It doesn't really change anything, does it?

Reply:
no, it just seems more logical to do the `oc apply` after the `cd` since this is where the file is located

Reply:
but I will approve the PR as-is, it's was just a minor comment

Reply:
yeah maybe it would look slightly better..